packager/VisibilityChecker.cc

```diff
@@ -833,28 +913,73 @@ vector<ast::ParsedFile> VisibilityChecker::run(core::GlobalState &gs, WorkerPool
             PropagateVisibility::run(gs, f);
         }
     }
-    auto result = VisibilityCheckerPass::run(gs, gs.packageDB(), workers, std::move(files));
+    auto result = VisibilityCheckerPass::run(gs, workers, std::move(files));
 
     if (gs.packageDB().genPackages()) {
-        Timer timeit(gs.tracer(), "visibility_checker.run.missing_imports_autocorrect");
-        UnorderedSet<core::packages::MangledName> affectedPackages;
+        Timer timeit(gs.tracer(), "visibility_checker.run.build_autocorrect");
+        UnorderedSet<core::packages::MangledName> packagesInEdit;
         for (auto &parsedFile : result) {
             auto pkgName = gs.packageDB().getPackageNameForFile(parsedFile.file);
             if (!pkgName.exists()) {
                 continue;
             }
-            affectedPackages.insert(pkgName);
+            packagesInEdit.insert(pkgName);
+        }
+
+        UnorderedMap<core::SymbolRef, vector<core::FileRef>> referencingFiles;
+        {
+            Timer timeit(gs.tracer(), "visibility_checker.run.build_autocorrect.build_referencingFiles");
```

Comment:
I don't know how our time series database deals with case sensitivity of timer names. Maybe let's stick with all lowercase names. Whether you want `referencingfiles` or `referencing_files` is up to you